pytorch_lightning/profiler/pytorch.py

```diff
@@ -327,8 +319,6 @@ def _init_kineto(self, profiler_kwargs: Any) -> None:
         self._schedule = ScheduleWrapper(schedule) if schedule is not None else schedule
         self._profiler_kwargs["schedule"] = self._schedule
 
-        activities = profiler_kwargs.get("activities", None)
```

Comment:
I would prefer to keep those. It is important users have control over the activities to be used.

Reply:
Yes, I just move the default value and user can also control over it though 'activities' argument.